webui/react/TODO.md

```diff
@@ -0,0 +1,13 @@
+# TODO
+
+- update latest master to allow localhost and github, circle ci, or all. deploy-aws-cluster
+- [x] set CI to set the server address env var
+- [x] artifact paths
+  - index. ./ colors.less
+  - det/dashboard paths
+  - replaced url
+  - hompage and PUBLICURL
+- [ ] use github actions to push artifacts as releases https://github.com/actions/create-release
+  - same for docs?
+- [ ] compute latest build URL and post to the PR
+- [ ] check streaming endpoints 
```

Comment:
I kind of like this centralized list of TODOs... This can be used to help define our tech debt backlogs...

Reply:
hmm we could maintain a file and use git to update it alongside whatever pr that's making related changes.